contracts/v2/PolygonRollupManager.sol

```diff
@@ -292,6 +293,17 @@ contract PolygonRollupManager is
         address indexed aggregator
     );
 
+    /**
+     * @dev Emitted when the trusted aggregator verifies a pessimistic proof
+     */
+    event VerifyPessimisticTrustedAggregator(
```

Comment:
same event helps the current bridge service to synchronize everything

Reply:
Fixed!